docs/ui.md

```diff
@@ -2,30 +2,32 @@
 title: Build the Connection Dialog with Connection Dialog v1
 ---
 
-The connection dialog prompts the user to enter connection and authentication information. That information is passed into the Connector Builder script to build the connection string. The dialog appears when creating a new connection or editing an existing connection and is used by both Tableau Desktop and Tableau Server.
-
-The connection dialog is mainly defined in the Tableau Custom Dialog (.tcd) file.
+## Define Tableau Custom Dialog file elements
```

Comment:
Do we want to have a note on this page that we recommend V2, or do we think that the message in the new connection-dialog page is sufficient?